### ISSUE TITLE
New Folder Structure

### ISSUE THREAD

**Issue by alanzchen** [conferency/find-my-reviewers#20]
We may need to restructure as our model extends:

```bash
# Renaming trained/ to models/

models
├── demo                                        # The name of the model
│   ├── author_vec_lib.json                     # The fitted vector library for authors
│   ├── demo.ldamodel                           # The main LDA models and supplemental files
│   ├── demo.ldamodel.dictionary
│   ├── demo.ldamodel.expElogbeta.npy
│   ├── demo.ldamodel.id2word
│   ├── demo.ldamodel.state
│   ├── demo.ldamodel.state.sstats.npy
│   ├── doc/                                    # Directory for dumped SpaCy doc objects that were used to trained the model 
│   ├── meta.sqlite                             # The metadata SQLite database of this model
│   └── paper_vec_lib.json                      # The fitted vectors for the documents
└── model_name2 # Another model
```

**Comment by alanzchen** [comment]
We may also deprecate the `lda_models.env` file.

**Comment by alanzchen** [comment]
Fixed in the latest PR #24
